### PR TITLE
Action sizes

### PR DESCRIPTION
**Pull request by ryan-roemer**
* Add `--action=sizes` report
* Add tsv output for all actions

/cc @ryanisinallofus @exogen @rgerstenberger @alexlande

### CODE REVIEW

File: lib/actions/sizes.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      return {
+        index: obj.index,
+        baseName: obj.baseName,
+        fileName: obj.fileName,
+        type: obj.isTemplate ? "template" : // eslint-disable-line no-nested-ternary
+          obj.isCode ? "code" :
+          "reference",
+        size: {
+          full: code.length,
+          min: obj.isCode ? minSize : code.length,
```
Comment (rgerstenberger) [inline lib/actions/sizes.js]: Not following this logic. Why would you show something other than `minSize` and `minGzSize` for these properties?

**Comment by rgerstenberger** [comment]
Minor comment. Rest LGTM!

File: lib/actions/sizes.js (hunk 1 above)
Comment (ryan-roemer) [inline lib/actions/sizes.js]: If the chunk is _not_ code, then is not actual code then it's a single reference (`123`) or array of references (`[123,456]`) which I've stringified as a replacement for "code" if this is a deduplicated reference.

In this case, the full / min / min+gz sizes are likely going to all be the same (or close enough in min+gz case)

**Comment by ryan-roemer** [comment]
Released in `inspectpack@0.5.0`

**Comment by ryan-roemer** [comment]
Released in `inspectpack@0.6.0`

**Pull request merged**